task-template.md

```diff
@@ -48,16 +48,16 @@ _Judging Criteria and Metrics_
 _Submission Procedure_ 
 * Submit your blog post following these steps:
  
-- [ ] [Sign Up](https://developer.algorand.org/accounts/signup/) or login on the Algorand Developer Portal.
+- [ ] [Sign Up](https://developer.algorand.org/accounts/signup/) or Log In onto the Algorand Developer Portal.
```

Comment:
```suggestion
- [ ] [Sign Up](https://developer.algorand.org/accounts/signup/) or log in on the Algorand Developer Portal.
```
Using capital letters in the middle of the sentence seems weird.
Also imo "to log in" (to use login feature) on a website is the correct usage. Maximum, it would be written separately.

Reply:
Both are correct in different ways, I just countered the inconcistency (Sign Up & login > Sign Up & Log In). Alternatively I would make it "Sign up ..." but the login on and login onto are both fine, I just tweaked it as I was already changing that sentence :) feel free to do whatever - changes should be permitted without me approving them AFAIK!

Reply:
Good to have you guys here.
I'm no expert in English grammar so I'll just take your suggestions. :)